index.html

```diff
@@ -0,0 +1,156 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <script src="https://cdn.tailwindcss.com"></script>
+  <script src="https://kit.fontawesome.com/b9a7923e8d.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    
+   <header class="border-b shadow-xl">
+    
+   
+    <div class="hidden md:flex justify-end gap-10 font-bold pb-[10px] pt-[10px] pr-[170px] text-white bg-[#272a31]">
+        <i class="fa-brands fa-facebook-f"></i>
+        <i class="fa-brands fa-twitter"></i>
+        <span>English</span>
+        <span>My Page</span>
+        <span>Logout</span>
+    </div>
+    <nav class="hidden md:flex items-center justify-between px-[12vw]  font-semibold">
+        <img class="w-[300px]" src="./assets/logo.png" alt="logo-image"/>
+        <ul class="flex items-center gap-10" >
+            <li class="text-orange-600"><a href="about.html">About</a></li>
+            <li><a href ="#">Program</a></li>
+            <li><a href ="#">Join</a></li>
+            <li><a href ="#">Sponsor</a></li>
+            <li><a href ="#">News</a></li>
+            <button class="ml-[4vw] p-2 border-4 border-orange-600">Fasion Wear</button>
+        </ul>
```

Comment:
- [ ] Good work implementing the navbar section here and your logo looks great👏  But it would be better if you make your logo clickable(redirects to home) and about page should have the same navbar as home page.